src/app/containers/ATIAnalytics/index.test.jsx

```diff
@@ -9,8 +9,20 @@ import { ServiceContextProvider } from '#contexts/ServiceContext';
 import ATIAnalytics from '.';
 import * as amp from './amp';
 import * as canonical from './canonical';
-import * as articleatiparams from './params/article';
-import * as frontpageatiparams from './params/frontpage';
+import { getPublishedDatetime, getCurrentTime } from '../../lib/analyticsUtils';
+
+jest.mock('../../lib/analyticsUtils', () => {
+  const utils = jest.requireActual('../../lib/analyticsUtils');
+
+  return {
+    ...utils,
+    getPublishedDatetime: jest.fn(),
+    getCurrentTime: jest.fn(),
+  };
+});
+
+getCurrentTime.mockImplementation(() => '00-00-00');
+getPublishedDatetime.mockImplementation(() => '1970-01-01T00:00:00.000Z');
```

Comment:
All this https://github.com/bbc/simorgh/pull/3559/files#diff-caaba44f5737f0c15f9a7b1d025b7041R12-R25

Can just be:

```js
import * as analyticsUtils from '../../../../lib/analyticsUtils';

analyticsUtils.getCurrentTime = jest.fn().mockReturnValue('00-00-00');
analyticsUtils.getPublishedDatetime = jest.fn().mockReturnValue('1970-01-01T00:00:00.000Z');
```

Reply:
Fixed!